htsohm/htsohm.py

```diff
@@ -247,6 +247,20 @@ def retest(m_orig, retests, tolerance, pseudo_material):
 
     session.commit()
 
+def get_all_parent_ids(run_id, generation):
+    sql = text(
+            (
+                'select parent_id from materials where run_id={0} and '
+                'generation={1};'.format(run_id, generation)
+            )
+        )
+    parent_ids = []
+    result = engine.execute(sql)
+    for row in result:
+        parent_ids.append(row[0])
+    result.close()
+    return parent_ids
+
```

Comment:
Session.query(Table.id).distinct()... did this in later commit? should have squashed it here.